### DISCUSSION TITLE
can't fetch content from the items api

### DISCUSSION THREAD

**Discussion by potros** [directus/directus#10527] [Archive]
when i try to access https://mydomain.com/items/:item    it works

but when i try to fetch the data iam unable to do that with this error

```
VM291:1  net::ERR_FAILED
(anonymous) @ VM291:1
VM291:2 Uncaught TypeError: Failed to fetch
    at <anonymous>:1:20
```

### To Reproduce

try to fetch the data from the api

### Errors Shown

```
VM291:1 GET https://mydomain.com/items/channels net::ERR_FAILED
(anonymous) @ VM291:1
VM291:2 Uncaught TypeError: Failed to fetch
    at <anonymous>:1:20
```

### What version of Directus are you using?

9.2.2

### What version of Node.js are you using?

17

### What database are you using?

sqlite

### What browser are you using?

chrome

### What operating system are you using?

linux

### How are you deploying Directus?

vps

**Comment by rijkvanzanten** [comment]
How are you trying to fetch the endpoint?

**Comment by potros** [comment]
fetch(domain.com/ items/:item).then(data => data.json())

**Comment by potros** [comment]
fetch(domain.com/ items/:item).then(data => data.json())

**Reply by rijkvanzanten** [reply]
Is that the exact javascript?

If so, there's a couple issues:

```js
fetch(domain.com/ items/:item).then(data => data.json())
```

You're missing the protocol (eg `https://` in front of `domain.com`), there's no quotes around the URL you're fetching, causing a syntax error in JS, and the `:item` will have to be the actual key you're trying to fetch.

**Comment by potros** [comment]
const data = await fetch("https://domain.xyz/items/channels").then(res => res.json())
VM296:1 GET https://domain.xyz/items/channels net::ERR_FAILED
(anonymous) @ VM296:1
VM296:2 Uncaught TypeError: Failed to fetch
    at <anonymous>:1:20

**Reply by rijkvanzanten** [reply]
Try adding a `catch` to that fetch to see what the error is. For what it's worth, a regular `fetch` call won't throw an error on 3xx-5xx responses from the API, so this error is most likely thrown in how fetch is being used in your environment, rather than from an error coming from the Directus API.

**Comment by potros** [comment]
TypeError: Failed to fetch
    at <anonymous>:1:20

**Comment by potros** [comment]
a removed the proxy server but the issue is the same

**Comment by rijkvanzanten** [comment]
I'm sorry, but there's simply not enough information and details on the infrastructure for us to really be of help 😔

At this point, I'm fairly certain the issue is caused by something other than Directus itself, but that's about all I can deduct from this.
